KlingonAssistant/data/mem-02-ch.xml

```diff
@@ -71,8 +71,8 @@
       <column name="_id">11665</column>
       <column name="entry_name">chabal</column>
       <column name="part_of_speech">n</column>
-      <column name="definition">something desired or requested</column>
-      <column name="definition_de"></column>
+      <column name="definition">wish, something desired or requested</column>
```

Comment:
Not really part of the definition, but it depends on how one interprets the english word "wish". Since "wish list" is translated as {chabal tetlh}, I think that the word "wish" should at least be there as a search term.